### PR TITLE
[MNT] Fix numba function caching the PR testing

### PR DESCRIPTION
**Pull request by MatthewMiddlehurst**
Missed a ! so the label is inverted, oops!

### CODE REVIEW

File: .github/workflows/pr_pytest.yml [hunk 1]
```diff
@@ -29,7 +29,7 @@ jobs:
         with:
           python-version: "3.10"
 
-      - if: ${{ github.event_name == 'pull_request' && contains(github.event.pull_request.labels.*.name, 'no numba cache') }}
+      - if: ${{ github.event_name == 'pull_request' && !contains(github.event.pull_request.labels.*.name, 'no numba cache') }}
```
Comment (chrisholder) [inline .github/workflows/pr_pytest.yml:32]: This is genius

**Review by chrisholder — APPROVED** [review]

**Comment by TonyBagnall** [comment]
maybe it needs a rerun to work?

**Comment by TonyBagnall** [comment]
nope, still skipping :(

**Comment by TonyBagnall** [comment]
hurrah!

**Review by TonyBagnall — APPROVED** [review]

**Pull request merged**